docs/doc_sources/physics_glossary.rst

```diff
@@ -199,10 +199,13 @@ Physics Glossary
   average_electron_density
     Volume-averaged electron density in the confined region :math:`\bar n_e`.
```

Comment:
I don't think it is necessary to change this definition, unless you think it makes it clearer or more consistent. @nelsonand

Reply:
It would make sense to call this `volume_averaged_electron_density`

Reply:
I agree with @tbody-cfs about this in principle, but for consistency we would have to propagate `volume_averaged_electron_temperature`, etc. through the code as well, which seems like it would just add more clutter everywhere. 

Can do if desired!

Reply:
I'd say I'm ambivalent.  Having the variable called `volume_averaged_electron_temperature` would be nicer, but it is a tedious job to do the renaming and the glossary correctly explains that this is the volume averaged electron density.

So @nelsonand, IMHO it's ok as is.